website/docs/reference/activation-strategies.md

```diff
@@ -2,130 +2,167 @@
 
-**Parameters**
+##### String operators
 
-- hostNames - _List of hostnames to enable the feature flag for._
+String operators accept multiple values and can be set to be case-sensitive or case-insensitive.
 
-This strategy has the following modelling name in the code:
+| Operator              | Description |
+|-------------------|------------------------------------------------|
+| `STR_CONTAINS`    | The context field contains any of the provided string values.       |
+| `STR_ENDS_WITH`   | The context field ends with any of the provided string values.     |
+| `STR_STARTS_WITH` | The context field starts with any of the provided string values.    |
 
-- **applicationHostname**
 
-## Multiple activation strategies {#multiple-activation-strategies}
+##### Versioning (SemVer) operators
 
-You can apply as many activation strategies to a flag as you want. When a flag has multiple strategies, Unleash will check each strategy in isolation. If any one of the strategies would enable the flag for the current user, then the flag is enabled.
+SemVer operators are used to compare version numbers such as application versions or dependency versions. SemVer operators only support single values.
 
-As an example, consider a case where you want to roll a feature out to 75% of your users. However, you also want to make sure that you and your product lead get access to the feature. To achieve this, you would apply a **gradual rollout** strategy and set it to 75%. Additionally, you would add a **user IDs** strategy and add `engineer@mycompany.com` and `productlead@mycompany.com`.
+The value must start with and contain at least major, minor, and patch versions. For example, `1.2.3`. Optionally, you can also define pre-release version information by adding a hyphen and series of full-stop separated identifiers after the patch version. For example, `1.2.3-rc.2`. Values with pre-release versions are considered less than versions without a pre-release in accordance with [the SemVer specification, item 11](https://semver.org/#spec-item-11).
 
-![A feature flag with two active strategies: a user ID strategy and a gradual rollout strategy. The strategies are configured as described in the preceding paragraph.](/img/control_rollout_multiple_strategies.png)
+| Operator        | Description           |
+|-------------|----------------------------------------------|
+| `SEMVER_EQ` | The context field is equal to the provided value.              |
+| `SEMVER_GT` | The context field is strictly greater than the provided value. |
+| `SEMVER_LT` | The context field is strictly less than the provided value.    |
 
-## Deprecated strategies
+#### Best practices
 
-### gradualRolloutUserId (DEPRECATED from v4) - Use Gradual rollout instead {#gradualrolloutuserid-deprecated-from-v4---use-gradual-rollout-instead}
+Server-side SDKs fetch the full feature flag configuration from Unleash, so every value that you add to that constraint value list increases the payload size.
```

Comment:
The full feature flag configuration matching your API key... I don't know how to phrase this correctly, but I think it's a good addition to mention this, because it hints our users that they can use the API keys to optimize the payload size by matching specific projects or environments